code/modules/antagonists/disease/disease_abilities.dm

```diff
@@ -428,6 +429,13 @@ new /datum/disease_ability/symptom/powerful/heal/youth
 
 /datum/disease_ability/symptom/powerful/inorganic_adaptation
 	symptoms = list(/datum/symptom/inorganic_adaptation)
+	
+//yogs start
+
+/datum/disease_ability/symptom/powerful/undead_adaption
+	symptoms = list(/datum/symptom/undead_adaption)
```

Comment:
![image](https://user-images.githubusercontent.com/20558591/49706633-4b2d8a80-fc27-11e8-97bf-e22711371730.png)

Reply:
that's weird because https://github.com/yogstation13/Yogstation-TG/blob/e94199f6b1bc08765e6ad9ad72b437e9c1e4b84a/code/datums/diseases/advance/symptoms/species.dm

Reply:
Its adaptation, not adaption

Reply:
ple ease giv me a g u n to shoot myself with